services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/northbound/switches/SwitchRulesSpec.groovy

```diff
@@ -114,6 +114,64 @@ class SwitchRulesSpec extends BaseSpecification {
         }
     }
 
+    @Unroll
+    def "Able to install default rules on a switch (install-action=#data.installRulesAction)"() {
+        assumeFalse("Unable to run the test because an OF_12 switch has one broadcast rule as the default",
+                srcSwitch.ofVersion == "OF_12" && data.installRulesAction != InstallRulesAction.INSTALL_BROADCAST)
+
+        given: "A switch without any rules"
+        def expectedRules = data.getExpectedRules(srcSwitch.dpId)
+
+        northbound.deleteSwitchRules(srcSwitch.dpId, DeleteRulesAction.DROP_ALL)
+        Wrappers.wait(RULES_DELETION_TIME) {
+            assert northbound.getSwitchRules(srcSwitch.dpId).flowEntries.empty
+        }
+
+        when: "Install rules on the switch"
+        def installedRules = northbound.installSwitchRules(srcSwitch.dpId, data.installRulesAction)
+
+        then: "The corresponding rules are really installed"
+        installedRules.size() == data.rulesInstalled
+        Wrappers.wait(RULES_INSTALLATION_TIME) {
+            compareRules(northbound.getSwitchRules(srcSwitch.dpId).flowEntries, expectedRules)
+        }
+
+        and: "Install missing default rules"
+        Wrappers.wait(RULES_INSTALLATION_TIME) {
+            northbound.installSwitchRules(srcSwitch.dpId, InstallRulesAction.INSTALL_DEFAULTS)
+            assert northbound.getSwitchRules(srcSwitch.dpId).flowEntries.size() == srcSwDefaultRules.size()
+        }
+
+        where:
+        data << [[
+                         installRulesAction: InstallRulesAction.INSTALL_DROP,
+                         rulesInstalled    : 1,
+                         getExpectedRules  : { switchId ->
+                             getRules(switchId).findAll { it.cookie == DefaultRule.DROP_RULE.cookie }
+                         }
+                 ],
+                 [
+                         installRulesAction: InstallRulesAction.INSTALL_BROADCAST,
+                         rulesInstalled    : 1,
+                         getExpectedRules  : { switchId ->
+                             getRules(switchId).findAll { it.cookie == DefaultRule.VERIFICATION_BROADCAST_RULE.cookie }
+                         }
+                 ],
+                 [
+                         installRulesAction: InstallRulesAction.INSTALL_UNICAST,
+                         rulesInstalled    : 1,
+                         getExpectedRules  : { switchId ->
+                             getRules(switchId).findAll { it.cookie == DefaultRule.VERIFICATION_UNICAST_RULE.cookie }
+                         }
+                 ],
+                 [
+                         installRulesAction: InstallRulesAction.INSTALL_DEFAULTS,
+                         rulesInstalled    : srcSwDefaultRules.size(),
+                         getExpectedRules  : { switchId -> getRules(switchId) }
+                 ]
+        ]
+    }
+
```

Comment:
Consider scenario:
1. switch with both default and flow rules
2. drop rules
3. call install rules (without flag)
expect: both default and flow rules are installed

Reply:
For now, it doesn't work. Only default rules are installed.